libpod/runtime_migrate.go

```diff
@@ -0,0 +1,47 @@
+package libpod
+
+import (
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+func (r *Runtime) migrate() error {
+	runningContainers, err := r.GetRunningContainers()
+	if err != nil {
+		return err
+	}
+
+	allCtrs, err := r.state.AllContainers()
+	if err != nil {
+		return err
+	}
+
+	logrus.Infof("stopping all containers")
```

Comment:
This looks like it can't be rolled back.  Do we want to add a "Are your sure you want to do this?" kind of prompt in here somewhere?

Reply:
I expect this to be used in a non interactive way, e.g. as part of the upgrade